Addressbook/src/module/search-user.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/ash
+
+db="../database/addressbook.db"
+
+usage_search() 
+{
+	echo "---------------------------------------"
+	echo "Search for User"
+	echo "Usage: [-u <user> ] [-e <email>] [-p <phone>]" 
+	echo "---------------------------------------"
+	exit 1 
+}
+
+search_user()
+{
+
+	match="cat $db"
+
+	while getopts "u:e:p:h" o; do
+	  case "$o" in
+	    u)
+		  name="${OPTARG}"
```

Comment:
The inconsistent indentation here makes it really hard to read the code.
I like the `awk` usage though, but would recommend not putting together a command and `eval`ing it like below, but running the appropriate commands explicitly.